### ISSUE TITLE
[FR] LV8728 support

### ISSUE THREAD

**Issue by jornada812** [MarlinFirmware/Marlin#14629]
I found and ordered drivers - lv8728, which work with 3.3 volts logic and can be used with more powerful motors, unlike lv8729. Do you need any changes in the firmware? What to write in the section steppers?

**Comment by RudyBenner** [comment]
They are supported

**Comment by jornada812** [comment]
> They are supported

but they are not in the list of drivers

**Comment by sl1pkn07** [comment]
> > They are supported
> 
> but they are not in the list of drivers

like the ST820

**Comment by RudyBenner** [comment]
My bad, not on the list of supported.

**Comment by sl1pkn07** [comment]
where is the list? you mean the list in the configure.h?

**Comment by jornada812** [comment]
Yes, in configuration.h. Steps for lv8729 are processed separately in stepper.h. And for lv8728? Is this actually the same driver, only the currents are different? Or am I wrong?

**Comment by sl1pkn07** [comment]
same question, but about ST820 comes with Fysetc AIO II board. what driver i need set?. the configuration/_adv for the board say A4988, but is correct?

greetings
